libraries/kafka_config.rb

```diff
@@ -34,7 +34,8 @@ class KafkaConfig < Chef::Resource
                     'kafka.request.logger' => 'WARN',
                     'kafka.controller' => 'TRACE',
                     'kafka.log.LogCleaner' => 'INFO',
-                    'state.change.logger' => 'TRACE'
+                    'state.change.logger' => 'TRACE',
+                    'kafka.authorizer.logger' => 'WARN'
```

Comment:
I am not opposed to this change, but for my own understanding, this is logger new to a specific version of Kafka? If so, we should consider making that the minimum version of the cookbook.